layouts/DocsPage/Navigation.tsx

```diff
@@ -118,7 +118,10 @@ export const getCurrentCategoryIndex = (
   categories: NavigationCategory[],
   href: string
 ) => {
-  const index = categories.findIndex(({ entries }) => hasSlug(entries, href));
+  const scopeLessHref = href.split("?")[0];
```

Comment:
Let's destructure this thing! also, "scopeless" is one word so no capital "L" needed.  
`const [ scopeLessHref ] = href.split("?")`
Your way is fine too, I've just been reading about destructuring :)